e2e/models/screenshot.css

```diff
@@ -0,0 +1,3 @@
+[data-test-id="__prosemirror_devtools_collapsed_button__"] {
```

Comment:
I seem to be still seeing the prosemirror dev tools (running with `npm run e2e:open`). If I inspect the snapshot then I see the element with this attribute but I don't see any sign of this CSS being applied.

Reply:
Hmm, that makes debugging tricky. Are the tests in `npm run e2e:open` working for you?

Reply:
Huh, had to refresh to get the video... That makes sense, as well as what you said about the snapshot failing as the system is different. So, only check snapshot tests on docker.